web/versioned_docs/version-0.13.0/advanced/deployment/cli.md

```diff
@@ -116,6 +116,21 @@ We need to do this to keep our CORS configuration up to date.
 
 That's it, your app should be available at `https://mycoolapp.com`! 🎉
 
+If you'd like to also access your app at `https://www.mycoolapp.com`, you can generate certs for the `www` subdomain. 
```

Comment:
This looks correct to me 👍 

Since we have docs versioning, there are multiple versions of this file:
- the main version (in the `web/docs/` folder)
- the 0.13.0 version (in the `web/versioned_docs/version-0.13.0`) <--- what you edited now
- the 0.12.0 version and so on

We should at least update the **main** and the **latest version** (in this case 0.13.0). Could you update the `cli.md` file for the main version as well?

Reply:
Yeah it is a bit convoluted, but itwas important we maintain multiple versions of docs and this seems to be the best way at the moment.

Btw, the easiest way to do this for you @Case-E is actually to just make these changes in `web/docs/`. That means they will be released with the next version of Wasp, but that is good enough, they don't have to urgently released. So I would advise going with just that, so you don't have to duplicate the changes.

If this is complex for you, let us know and we someone from the team can take it over and move them from 0.13 to the `web/docs` for you, no worries.

Reply:
Thanks @Martinsos! That totally makes sense to have versioned docs for your use-case. 

Not too important but it would be great if the edit this page link brought us to the `web/docs` instead of the versioned docs. I did this totally on the GitHub UI so didn't notice where I was suggesting the change. However, now that I am typing this, this may not work in all cases. `web_docs` would only have the latest version while users might want to edit older versions of the docs too. 🤷

<img width="214" alt="image" src="https://github.com/wasp-lang/wasp/assets/25521878/68270026-3a09-469a-884f-017b30550ec3">

I've made the changes now for this PR :)